### PR TITLE
Sort elements

### PR DESCRIPTION
**Pull request by franosincic**
Registry initialization doesn't guarantee same order of elements every time, so on adding element/assessment elements will be sorted by position in configuration file.

### CODE REVIEW

**Comment by vladimyr** [comment]
@underscope Is this ok to get merged?

**Comment by underscope** [comment]
I communicated with @franosincic desired order.

File: client/components/common/tce-core/AddElement/SelectAssessment.vue [hunk 1]
```diff
@@ -30,7 +31,7 @@ export default {
   },
   computed: {
     rows() {
-      return chunk(this.elements, this.rowSize);
+      return chunk(sortBy(this.elements, 'position'), this.rowSize);
```
Comment (underscope) [inline client/components/common/tce-core/AddElement/SelectAssessment.vue]: Might make sense to do the sorting on registry side upon import.

File: client/ElementRegistry.js [hunk 2]
```diff
@@ -18,12 +18,16 @@ export default class ElementRegistry {
...
+    });
     const extensions = await this.loadExtensionList();
-    await Promise.map(extensions, location => this.load(location, true));
+    await Promise.map(extensions, (location, position) => {
+      return this.load(location, elementList.length + position, true);
+    });
   }
 
-  async load(location, isExtension) {
+  async load(location, position, isExtension) {
```
Comment (underscope) [inline client/ElementRegistry.js]: Move isExtension and position to options object. Set position to last as a default.

**Pull request merged**